selenium_task_1.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class DuckDuckGoSearch(unittest.TestCase):
+    def setUp(self):
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument('--no-sandbox')
+        chrome_options.add_argument('--lang=pl')
+        self.driver = webdriver.Chrome(options=chrome_options)
+
+    def tearDown(self):
+        self.driver.quit()
+
+    def test_search_in_duck_duck_go(self):
+        driver = self.driver
+        driver.get("http://www.duckduckgo.com")
+        field = driver.find_element_by_xpath('//*[@id="search_form_input_homepage"]')
```

Comment:
It is not a good idea to use **//*** as an element. If we know what is the type of this element - we should specifiy it. For this situation - should be input - so it should be `//input[@id="search_form_input_homepage"]` .
If the id is specified and unique enough - we can use  `driver.find_element_by_id('search_form_input_homepage')` instead of xpath.

Reply:
To have more readable code - the name should be specified. For this one - `search_input` will be better than `field`.